### PR TITLE
W-14374222-rns bot 1.4.0

### PR DESCRIPTION
**Pull request by tasmoore**

### CODE REVIEW

**Review by ms-dwagner — APPROVED** [review]

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.4.0-release-notes.adoc [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+RPA Bot is a software robot that runs in an on-premise environment and executes the assigned automations.
+
+include::partial$release-notes/rn-last-version-message.adoc[] 
+
+== November , 2023
+
+// == What's New
+include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]
+
+* The script now provides the Bot GUID when executing a workflow so that the Bot GUID can be provided via runtime variable in RPA Builder.
```
Comment (dukesphere) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.4.0-release-notes.adoc]: "via runtime ..." -> "via the..." or "via a runtime"?

Comment (dukesphere) [reply rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.4.0-release-notes.adoc]: Do the first 2 bullets need to be parallel to the last ones that start with the verb?

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.4.0-release-notes.adoc [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* The *Check interval* option is now constrained to a range between 1 and 15 seconds. If you currently have the option set to a value higher than 15 seconds, then the value will be reset to 15 seconds upon release of this enhancement.
+* Added a new *System Settings* panel to the RPA Bot Configurator. This panel provides options to configure basic settings on the host computer.
+* Extended the logging system information by adding `key`, `type`, and `name` registry values.
+* Removed the *Test Settings* button from the Session Settings panel.
+* Created a command line test tool that is similar to FreeRdp but uses the MuleSoft `rpa-freerdp-2.10.dll`.
```
Comment (dukesphere) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.4.0-release-notes.adoc]:
```suggestion
* Created a command-line test tool that is similar to FreeRdp but uses the MuleSoft `rpa-freerdp-2.10.dll`.
```

do we hyphenate when using as an adjective?

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.4.0-release-notes.adoc [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+1]
+
+[%header,cols="2a,1a"]
+|===
+| Issue Resolution | ID
+|The `RegisterProcessInstance` now functions properly as a result of increasing the number of attempts when starting a RunActivity to 20x2 seconds.| W-14283854
+|RPA Bot no longer stops executing processes. | W-14202513
+|The runtime for processes that are executed more than once by RPA Bot is no longer different. | W-14159982
```
Comment (dukesphere) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-bot-1.4.0-release-notes.adoc]: Not sure if it matters, but different than what?

**Review by dukesphere — APPROVED** [review]

**Review by dukesphere — APPROVED** [review]

**Pull request merged**